packages/test/test-service-load/testConfig.json

```diff
@@ -27,18 +27,13 @@
 			},
 			"optionOverrides": {
 				"odsp": {
-					"configurations": {
-						"Fluid.Summarizer.ValidateSummaryBeforeUpload": [true, false],
-						"Fluid.Summarizer.UseDynamicRetries": [true, false],
-						"Fluid.Summarizer.PendingOpsRetryDelayMs": [true, false]
-					}
-				},
-				"odsp-odsp-df": {
```

Comment:
FYI for reviewers: "odsp" is a superset over "odsp-<driver end point>", unless later exists